Muhire-Josue/freelancerz
# feature(recommend developers)

#### What does this PR do?

- It allows the Client to view applications with recommendation

#### Description of Task to be completed?

- When the client view application to his/her job he should be able to see a recommendation for every application

#### How should this be manually tested?

- Clone: `git clone https://github.com/Muhire-Josue/freelancerz.git`

- Install dependencies: `npm install`

- Check into this branch: `ft-recommend-developers-#173949462`

- Run the server: `npm run dev` ==> should get this message in the terminal `Listening on port 3000`

- Hit this URL in postman to disable email notifications `PUT http://localhost:3000/api/job/applications`

- Send this data
```
{
     "id": jobId
}
```

#### Any background context you want to provide?

- make sure you have [node v10 - v13](https://nodejs.org/en/download/) not node v14

- make sure you have [Postgres](https://www.postgresql.org/download/) up and running

- create .env file and use the .env.example file as a reference to create the variable environment needed for this project
#### What are the relevant pivotal tracker stories?

- [173949462](https://www.pivotaltracker.com/story/show/173949462)

#### Screenshots (if appropriate)
![viewAllApplications](https://user-images.githubusercontent.com/37122177/89119642-35ca3b80-d4b0-11ea-84e6-8c21cc59a546.PNG)
![viewAnApplication](https://user-images.githubusercontent.com/37122177/89119644-3793ff00-d4b0-11ea-8c3d-adb1020e4885.PNG)
